### PR TITLE
yolov8-pose integration

### PR DESCRIPTION
**Pull request by bstandaert**
Added the bottom-up module using yolov8-pose from ultralytics.

**Needed to upgrade the ultralytics version from "8.0.61" to "8.0.100"**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -17,7 +17,7 @@ lightning = "^2.0"
 pytorch_lightning = "^2.0"
 numpy = "^1.23.5"
 openmim = "^0.3.9"
-ultralytics = "8.0.61"
+ultralytics = "8.0.100"
```
Comment (victorjoos) [inline pyproject.toml:20]: If you have poetry, could you run "poetry update", to update the "poetry.lock" file ? Otherwise I'll run it on my side

**Review by victorjoos — APPROVED** [review]
Perfect ! I'll just run the "poetry lock --update" and add the poetry.lock change to the PR, and then we can merge !

File: pyproject.toml (hunk 1 above)
Comment (victorjoos) [reply pyproject.toml:20]: Actually better to run "poetry lock --no-update", to only update this dependency.

Comment (bstandaert) [reply pyproject.toml:20]: I am using anaconda. Old school 🥸

**Pull request merged**